config/locales/account/en.yml

```
@@ -65,6 +65,15 @@ en:
         enter your email and password again.
       piv_cac: Sign in with your government employee ID
       tab_navigation: Account creation tabs
+    login_exit:
+      account_page: account page
+      bullet1: You won’t be able to use %{app_name} to access %{sp_name} until you
+        share your information with them.
+      bullet2_html: You’ll still have a %{app_name} account which you can manage or
+        delete on your %{link_html}.
+      exit: Exit %{app_name}
+      header_html: <p>If you exit %{app_name} now:</p>
```

Rather than embed the paragraph in the HTML, I'd expect it to be part of the view. But it looks like we already have a paragraph in the view markup as well, so we probably don't need it at all?

```suggestion
      header_html: If you exit %{app_name} now:
```